app.R

```diff
@@ -110,29 +114,41 @@ ui <- navbarPage("Formula 1 Dashboard",
                  tabPanel('Race Information',
                           fluidRow(
                             # Dropdown for grand prix
-                            column(6, selectInput(inputId = 'gp',
-                                                  label = 'Choose Grand Prix',
-                                                  choices = unique(race_results$GP),
-                                                  selected = "Bahrain Grand Prix",
-                            )),
+                            column(3,
+                                   fluidRow(column(
+                                     12,
+                                     selectInput(
+                                       inputId = 'gp',
+                                       label = 'Choose Grand Prix',
+                                       choices = unique(race_results$GP),
+                                       selected = "Bahrain Grand Prix",
+                                     )
+                                   )),
+                                   fluidRow(column(
+                                     12,
+                                     align="center",
+                                     imageOutput("track_layout")
```

Comment:
We can set the height of `imageOutput` to 200px (by passing in height="200px") since the images of the tracks all have heights of under 200 pixels. This will remove a lot of the blank spaces between the track image and the information below it.
Before:
![image](https://user-images.githubusercontent.com/51040875/222838564-ad222f18-0721-4df3-98ef-e227bedc22c9.png)


After:
![image](https://user-images.githubusercontent.com/51040875/222838057-6642f21c-358b-4ebe-974d-b88512bb3070.png)

Reply:
I was struggling how to fix this and thought I tried this, thanks! Will fix